example/main.c

```diff
@@ -22,7 +21,7 @@
 #include <stdlib.h>
 #include <sys/mman.h>
 
-#include "sha3.h"
+#include "sha3iuf/sha3.h"
```

Comment:
I don't see a need to add a directory here. It will break other projects

Reply:
I can remove that. I like the convention of `#include "library/important_file.h"`